services/src/services/errorTelemetry/errorTelemetry.service.js

```diff
@@ -7,16 +7,34 @@
  * simply be taking the last of version records associated with the instance.
  */
 
-const createService = require('feathers-sequelize');
-const createModel = require('../../models/error_log');
+const fs             = require('fs');
+
 const hooks = require('./errorTelemetry.hooks');
 
 module.exports = function (app) {
-  const options = {
-    name: 'error_log',
-    Model: createModel(app)
-  };
+  class MyService {
+    constructor() {
+    }
+    create(data) {
+      // Should be impossible because it passed the hooks step
+      if(!data) {
+        return Promise.reject({status:'KO'});
+      }
+
+      this.storeData(data);
+
+      return Promise.resolve({status:'OK'});
+      // Called
+    }
+
+
+    storeData(data) {
```

Comment:
Why is this in a separate function? I don't anticipate any other methods hitting the telemetry service are going to need to `storeData`?